openai/index.ts

```diff
@@ -38,12 +37,28 @@ export class OpenAIApi extends originalOpenAI.OpenAIApi {
   ) {
     // return super.createChatCompletion(createChatCompletionRequest, options);
     const res = await this.cbClient.createCompletion({
-      variables: createChatCompletionRequest.variables || {},
       userId: createChatCompletionRequest.user,
       chatContext: {
         messages: createChatCompletionRequest.messages,
       },
       projectId: createChatCompletionRequest.projectId,
+      prompt: "\n", // we need to set prompt to avoid the default prompt
```

Comment:
Again, perhaps "define prompt in case you manage your prompts in the code"

Reply:
This is just a comment for us. If users are using the openai drop-in replacement we assume that they already manage their own prompts.